dashboard/static/css/home.css

```diff
@@ -0,0 +1,3 @@
+form#add-push-application input[type="submit"] {
+    display: block;
+}
```

Comment:
We could! That would make some things easier in the short term. At the same time, it doesn't cost much to have a separate stylesheet for the homepage, and we did spend a lot of time moving stuff out of that big monolithic *mdn-screen.css* file on MDN a while back.